spec/services/middleware_topology_service_spec.rb

```diff
@@ -29,80 +29,86 @@
     it "topology contains the expected structure and content" do
       allow(middleware_topology_service).to receive(:retrieve_providers).and_return([ems_hawkular])
 
-      middleware_deployment1 = MiddlewareDeployment.create(:ext_management_system => ems_hawkular, :middleware_server => middleware_server,
-                                                           :ems_ref               => long_id_2,
-                                                           :name                  => "hawkular-wildfly-agent-download.war",
-                                                           :nativeid              => "Local~/deployment=hawkular-wildfly-agent-download.war")
+      md_deployment1 = MiddlewareDeployment.create(:ext_management_system => ems_hawkular,
+                                                   :middleware_server     => md_server,
+                                                   :ems_ref               => long_id_2,
+                                                   :name                  => "hawkular-wildfly-agent-download.war",
+                                                   :nativeid              => "Local~/deployment=hawkular-wildfly-"\
```

Comment:
@Jiri-Kremser Please don't escape the EOL with `\`, use shorter names or let the line be a little longer.